src/path-optimization/simple-time-parameterization.cc

```diff
@@ -30,242 +30,213 @@
-        vector_t v_inv (robot->numberDof());
-        for (std::size_t i = 0; i < input->numberPaths(); ++i) {
-          PathPtr_t p = input->pathAtRank(i);
-          interval_t paramRange = p->paramRange();
-          p->timeParameterization (TimeParameterizationPtr_t(), paramRange);
-
-          // Compute B
-          p->velocityBound (v, paramRange.first, paramRange.second);
-          v_inv = (v.array() == 0).select(infinity, v.cwiseInverse());
-          const value_type B = std::min(
-              ( ub.cwiseProduct(v_inv)).minCoeff(),
-              (-lb.cwiseProduct(v_inv)).minCoeff());
-          if (B <= 0 || B != B) {
-            HPP_THROW(std::runtime_error,"Invalid parametrization derivative "
-                "velocity bound: " << B);
-          }
-
-          // Compute the polynom and total time
-          value_type T;
-          TimeParameterizationPtr_t tp;
-          switch (order) {
-            case 0:
-              tp = computeTimeParameterizationFirstOrder (paramRange.first, paramRange.second, B, T);
-              break;
-            case 1:
-              tp = computeTimeParameterizationThirdOrder (paramRange.first, paramRange.second, B, T);
-              break;
-            case 2:
-              tp = computeTimeParameterizationFifthOrder (paramRange.first, paramRange.second, B, maxAcc, T);
-              break;
-            default:
-              throw std::invalid_argument ("Parameter SimpleTimeParameterization/order should be in { 0, 1, 2 }");
-              break;
-          }
-
-          checkTimeParameterization (tp, order, paramRange, B, maxAcc, T);
-
-          PathPtr_t pp = p->copy();
-          pp->timeParameterization (tp, interval_t (0, T));
-
-          output->appendPath (pp);
-        }
-        return output;
-      }
-
-      SimpleTimeParameterization::SimpleTimeParameterization (const Problem& problem):
-        PathOptimizer(problem) {}
-
-      // ----------- Declare parameters ------------------------------------- //
-
-      HPP_START_PARAMETER_DECLARATION(SimpleTimeParameterization)
-      Problem::declareParameter(ParameterDescription (Parameter::FLOAT,
-            "SimpleTimeParameterization/safety",
-            "A scaling factor for the joint bounds.",
-            Parameter(1.)));
-      Problem::declareParameter(ParameterDescription (Parameter::INT,
-            "SimpleTimeParameterization/order",
-            "The desired continuity order.",
-            Parameter((size_type)0)));
-      Problem::declareParameter(ParameterDescription (Parameter::FLOAT,
-            "SimpleTimeParameterization/maxAcceleration",
-            "The maximum acceleration for each degree of freedom. Not considered if negative.",
-            Parameter((value_type)-1)));
-      HPP_END_PARAMETER_DECLARATION(SimpleTimeParameterization)
-    } // namespace pathOptimization
-  } // namespace core
-} // namespace hpp
+namespace core {
+namespace pathOptimization {
+using std::isfinite;
```

Comment:
I would prefer to split apart re-indentation and actual changes.

Reply:
Eeek, I didn't even see that, thanks.